ADO.NET/ResultSetPrinter.cs

```diff
@@ -2,65 +2,103 @@
 using System.Configuration;
 using System.Data.Common;
 using Vertica.Data.VerticaClient;
+using System.Collections.ObjectModel;
+using System.Linq;
 
-class ResultSetPrinter {
+internal class ResultSetPrinter {
+        private int m_maxColumnWidth = Int32.Parse (ConfigurationManager.AppSettings["MaxColumnWidth"]);
+        private int m_widestColumnRead = 0;
+        private VerticaDataReader m_reader;
+        private ReadOnlyCollection<DbColumn> m_columns;
+        private const string TRUNCATION_SYMBOL = "...";
```

Comment:
nit: put const declarations first in the list

Reply:
Good point.  I did that and cleaned up the member declarations.